Dockerfile

```diff
@@ -19,21 +20,17 @@ RUN swift package resolve
 COPY . .
 
 # Compile with optimizations
-RUN swift build \
-	--enable-test-discovery \
-	-c release \
-	-Xswiftc -g
+RUN swift build --enable-test-discovery -c release
```

Comment:
@gwynne why did you remove the `-Xswiftc -g` flag? That's required for all the backtrace stuff to work

Reply:
It's automatically included even in release builds since Swift 5.2, not sure if it was a point release that made that particular change but I know it's been the case since at least 5.2.2. Add `-v` to show the compiler invocations if you want to verify, I certainly wouldn't mind the confirmation.